tests/cases/fourslash/incrementalJsDocAdjustsLengthsRight.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference path="fourslash.ts" />
+
+// @noLib: true
+////function camelcase(flag) {
+//// return flag.split('-').reduce(function (str, word) {
+//// return str + word[0].toUpperCase() + word.slice(1);
+//// });
+////}
+////
+/////**
+//// * Pad `str` to `width`.
+//// *
+//// * @param {String} str
+//// * @param {Number} wid/*1*/
+goTo.marker('1');
+edit.insert("th\n@");
```

Comment:
Is there anything worth verifying here beyond "it doesn't crash"? I don't see it checking any lengths have been adjusted right (per the name).

Reply:
Just the not crashing. The position being wrong causes crashing.

Reply:
I get that was the issue being addressed. I'm just suggesting that if you're going to add a fourslash test (which aren't cheap to run), might as well make it more useful and actually verify the JsDoc tags/spans also. It's a pretty narrow/specific test as-is that adds little value once this specific bug is fixed.